packages/preset-wind/src/rules/behaviors.ts

```diff
@@ -1,22 +1,22 @@
 import type { Rule } from '@unocss/core'
 import { colorResolver, handler as h } from '@unocss/preset-mini/utils'
 
-const listStyleProps = ['disc', 'circle', 'square', 'decimal', 'zero-decimal', 'greek', 'roman', 'upper-roman', 'alpha', 'upper-alpha']
-
 export const listStyle: Rule[] = [
-  [new RegExp(`^list-((?:${listStyleProps.join('|')})(?:-outside|-inside)?)$`), ([, value]) => {
-    const style = value.split(/-outside|-inside/)[0]
-    const position = /outside|inside/.exec(value) ?? []
-
-    if (position.length) {
+  // base
+  [/^list-(disc|circle|square|decimal|zero-decimal|greek|roman|upper-roman|alpha|upper-alpha)(?:-(outside|inside))?$/, ([, style, position]) => {
```

Comment:
Why do we do this refactoring? Static regex is more efficient then dynamic generated one

Reply:
The constructor is removed and changed with inlined union `|`. Did I misuse the syntax?

Reply:
Oh sorry I mis read the diff in wrong direction 😅😅